index.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="eng">
+  <head>
+    <link rel="stylesheet" href="indexStyles.css" />
+    <link rel="stylesheet" href="homeStyles.css" />
+  </head>
+  <body>
+    <!--Header Section-->
```

Comment:
while I appreciate these descriptive comments, usually the best practice is that ~95% (or something like that) of code should be readable without comments. in future, you don't need to put these kinds of descriptive comments unless if there's something that an external user likely wouldn't be able to discern without the additional context